### DISCUSSION TITLE
Feedback for "Tutorials"

### DISCUSSION THREAD

**Discussion by anjrotDev** [joe-bell/cva#318] [Feedback]
Hi my name is Antonio Rodríguez

I wanted to share some feedback regarding Class Variance Authority. While it's an incredibly useful tool, I noticed there’s very little information available about CVA in Spanish. To help bridge that gap, I created a video explaining how to use it as part of a series on building a component library.

Here’s the link to the video: [Class Variance Authority (CVA) | Serie de Librería de Componentes](https://youtu.be/WCLdicJfkSc)

It would be amazing if you could check it out and share it with others who might find it helpful. I believe it could be a great resource for Spanish-speaking developers looking to learn more about CVA.

Thanks for your time and for creating such an awesome tool!

Best regards,
AnjrotDev

**Comment by joe-bell** [comment]
Thanks so much for doing this @anjrotDev

Would love to include this in [beta.cva.style/tutorials](https://beta.cva.style/tutorials)

**Reply by anjrotDev** [reply]
I'm glad you liked it! It would be awesome if you could post and share my videos. Thank you so much for considering it, I really appreciate your support!
